packages/asc-ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -15,8 +16,13 @@ const Checkbox: React.FC<Props & React.HTMLAttributes<HTMLDivElement>> = ({
   error,
   ...otherProps
 }) => {
-  const [checked, setChecked] = React.useState(!!checkedProp)
-  const [focus, setFocus] = React.useState(false)
+  const [checked, setChecked] = useState(!!checkedProp)
+  const [focus, setFocus] = useState(false)
+  const { setActive } = useContext(LabelContext)
+
+  useMemo(() => {
```

Comment:
Looks like useMemo / setActive is causing a build error:
 
![image](https://user-images.githubusercontent.com/7781865/67382719-4008f200-f58e-11e9-9897-926d0f2bbca1.png)